tee-worker/omni-executor/intent/executors/cross-chain/src/single_chain.rs

```diff
@@ -112,55 +94,55 @@ impl<BinanceClient: BinanceApi, SolanaClient: SolanaClientTrait>
 	}
 
 	// call backend API in one step - it requires backend has signing access to signer, which will be gradually deprecated
-	async fn pumpx_do_market_order(
-		&self,
-		intent_id: IntentId,
-		amount: String,
-		token_ca: String,
-		recipient_address: String,
-		access_token: &str,
-		pumpx_config: &PumpxConfig,
-	) -> Result<Vec<u8>, ()> {
-		debug!("executing pumpx_do_market_order");
-		let body = CreateMarketOrderTxBody {
-			request_id: intent_id,
-			chain_id: pumpx_config.to_chain_id,
-			token_ca,
-			swap_type: match pumpx_config.swap_type {
-				1 => SwapType::Buy,
-				2 => SwapType::Sell,
-				_ => {
-					log::error!("Unsupported swap type: {}", pumpx_config.swap_type);
-					return Err(());
-				},
-			},
-			amount_in: amount,
-			double_out: pumpx_config.double_out,
-			is_one_click: pumpx_config.is_one_click,
-			address: recipient_address,
-			is_anti_mev: pumpx_config.is_anti_mev,
-			is_auto_slippage: pumpx_config.is_auto_slippage,
-			gas_type: match pumpx_config.gas_type {
-				1 => GasType::Slow,
-				2 => GasType::Medium,
-				3 => GasType::Fast,
-				_ => {
-					log::error!("Unsupported gas type: {}", pumpx_config.gas_type);
-					return Err(());
-				},
-			},
-			slippage: pumpx_config.slippage,
-			wallet_index: pumpx_config.wallet_index,
-		};
-		debug!("Calling pumpx create_market_order_tx, body: {:?}", body);
-		let response =
-			self.pumpx_api.create_market_order_tx(access_token, body).await.map_err(|_| {
-				log::error!("Failed to create market order tx");
-			})?;
-
-		debug!("Response create_market_order_tx: {:?}", response);
-		Ok(response.encode())
-	}
+	// async fn pumpx_do_market_order(
```

Comment:
Are we planning to use this later? Can this be removed?

Reply:
I'm keeping it uncommented for now with `clippy(allow(unused))` derive macro to not get complains from `cargo clippy`, I'm keeping it around as i'm not fully sure if we may or may not need it later.